### ISSUE TITLE
Parsing failed with characters like 'ë'

### ISSUE THREAD

**Issue by wzxu** [zhaocai/alfred2-google-workflow#4]
Just try typing 'g ë' and immediately Alfred reports a fatal error.

Checking in the log it reads:

```
Exception parsing
Line: 1
Position: 743
Last 80 unconsumed characters:
<suggestion data="ë"/>
```

**Comment by zhaocai** [comment]
cannot fix it easily. It is raised from internal library `rexml`. do you use this kind of characters very often?

**Comment by wzxu** [comment]
Not really. Just happened to run into one today rather unexpectedly. Thought I'd report.

**Comment by zhaocai** [comment]
Thanks. A better solution is to just google api instead of raw http request. 

A todo task is to add google search for image, book, etc.

**Comment by petrnagy** [comment]
Actually, characters like "ěščřžýáíéůú" are very common in some European languages.
